lib/fastlane/plugin/sentry/actions/sentry_upload_sourcemap.rb

```diff
@@ -79,7 +81,14 @@ def self.available_options
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :url_prefix,
                                        description: "Sets a URL prefix in front of all files",
-                                       optional: true)
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :app_identifier,
+                                      short_option: "-a",
+                                      env_name: "SENTRY_APP_IDENTIFIER",
+                                      description: "App Bundle Identifier",
+                                      optional: true,
+                                      default_value: CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier))
```

Comment:
It looks like adding a default value for this is a breaking change, as all releases version names will now be prefixed by the app identifier, which would require a code change in the JS to match. I think it's probably best to not have a default_value here.